.github/codecov.yml

```diff
@@ -8,4 +8,5 @@ coverage:
       default:
         threshold: 5%
 ignore:
-  - internal
+  - examples
+  - scripts
```

Comment:
just wondering why not just call it build instead of scripts ?

Reply:
I remember some C and Python projects used build/ for target directory with artifacts and temp files.

And dir like scripts/ is a part of a wider convention for project automation https://github.blog/engineering/scripts-to-rule-them-all/

Reply:
`examples/` and `scripts/` are also pretty common in Go projects.

Though not official, the https://github.com/golang-standards/project-layout recommends this pattern.